nengo/utils/testing.py

```diff
@@ -436,7 +434,7 @@ def __init__(self, n_threads):
         for t in threads:
             t.join()
             if not t.assertion_result:
-                reraise(*t.exc_info)
+                raise self.exc_info[0].with_traceback(*self.exc_info[1:])
```

Comment:
Is this not a bug? I think this should be:
```python
    raise self.exc_info[1].with_traceback(self.exc_info[2])
```
to be consistent with `compat.py`:
```python
    def reraise(tp, value, tb):
        raise value.with_traceback(tb)
```
and https://docs.python.org/3/library/exceptions.html#BaseException.with_traceback.

Reply:
Done